Fix for T206766523 ("Your diff, D65462767, broke some tests")

fduwjj opened the pull request:
Summary:
This is trying to fix a regression caused by https://github.com/pytorch/pytorch/pull/139757. We now don't want to log args and kwargs directly because if they contain tensor or tensor subclass it would take lots of time in conversion to string or even not supported. 

Reviewed By: fduwjj

Differential Revision: D65490202




cc @H-Huang @awgu @kwen2501 @wanchaol @fegin @wz337 @wconstab @d4l3k @c-p-i-o

fduwjj commented:
This is a auto generated diff trying to fix a internal test failure. ([T206766523](https://www.internalfb.com/intern/tasks/?t=206766523)

XilunWu commented on torch/_logging/_registrations.py:
```diff
@@ -135,7 +135,7 @@
 register_artifact(
     "not_implemented",
-    "Prints log messages whenever we return NotImplemented in a multi-dispatch, letting you trace through each object we attempted to dispatch to",
+    "Prints log messages whenever we return NotImplemented in a multi-dispatch, letting you trace through each object we attempted to dispatch to"
```
Is this suggested by the internal linter? This is contradictory to Github CI.

fduwjj replied:
it is auto-generated by agent.

XilunWu reviewed (approved):
we can land if internal testing is green.

ezyang reviewed (changes requested):
I don't see why this right. In particular, there are plenty of other artifacts. Why is not_implemented the only one to be reregistered? This can't be the correct fix.

Commit:
Fix for T206766523 ("Your diff, D65462767, broke some tests") (#139804)

Summary:

This diff was created by the [Meta Engineering Agent](https://fb.workplace.com/groups/1205375940786433/permalink/1205378807452813/) to automatically fix the broken tests assigned to you / your team in T206766523:
- The agent searched for and edited the files it deemed relevant for fixing the tests
- After generating the patch, the broken test(s) were rerun to verify they now pass
- Regression tests were also run to verify CI still passes

---

**If this diff is the correct fix for T206766523, please accept the diff**. *Note that while we set a high quality bar for the agent publishing these diffs, LLM-powered agents still make mistakes, so your critical review is much appreciated and will help us improve!*

**If changes need to be made:** Please commandeer the diff, or comment with the specific changes needed.

---

*P.S. If you have broader feedback, please post in the [Meta Engineering Agent - Users group](https://fb.workplace.com/groups/1205375940786433)! We would love to hear from you on improving the agent to better assist you in your daily work.*

Test Plan:
```
buck2 test 'fbcode//mode/opt' fbcode//inference_enablement/model_processing/tests:ien_publish_in_trainer_tests -- --exact 'inference_enablement/model_processing/tests:ien_publish_in_trainer_tests - test_sp_in_trainer_publish (inference_enablement.model_processing.tests.ien_publish_in_trainer_tests.IENPublishInTrainerTests)'
```
https://www.internalfb.com/intern/testinfra/testrun/10977524148369514

Differential Revision: D65490202
